internal/config/plugin_installer.go

```diff
@@ -665,7 +670,7 @@ func PluginCommand(out io.Writer, cmd string, args []string) {
 		for _, plugin := range args {
 			// check if the plugin exists.
 			for _, p := range Plugins {
-				if p.Name == plugin && p.Default {
+				if (p.Name == plugin && p.Builtin) || p.Name == "initlua" {
 					fmt.Fprintln(out, "Default plugins cannot be removed, but can be disabled via settings.")
```

Comment:
If we are renaming "default" to "built-in" all over the place anyway, don't we update to update this log message as well?